src/ontology/mondo-edit.obo

```diff
@@ -586047,6 +586047,16 @@ property_value: http://purl.org/dc/elements/1.1/date "2024-07-03T14:16:10Z" xsd:
 property_value: http://purl.org/dc/terms/creator https://orcid.org/0000-0002-0587-4693
 property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/7719" xsd:anyURI
 
+[Term]
+id: MONDO:1040008
+name: CAMK2D-related neurodevelopmental disorder and dilated cardiomyopathy
+def: "A neurodevelopmental disorder caused by variation in the CAMK2D gene. This disorder is characterised by intellectual disability, speech and motor delay, behavioural problems and dilated cardiomyopathy. Patients often present brain structural anomalies and hypotonia, and less frequently, seizures." [https://orcid.org/0000-0002-0587-4693, https://orcid.org/0000-0002-0987-9862, PMID:37372357, PMID:38272033]
+is_a: MONDO:0100500 ! Mendelian neurodevelopmental disorder
+intersection_of: MONDO:0100500 {source="PMID:37372357", source="PMID:38272033", source="https://orcid.org/0000-0002-0587-4693"} ! Mendelian neurodevelopmental disorder
```

Comment:
We usually don't add a source for the logical definitions. I will move these to the SubClassOf annotation